phpBB/phpbb/passwords/driver/base.php

```diff
@@ -13,15 +13,15 @@
 
 namespace phpbb\passwords\driver;
 
-abstract class base implements driver_interface
+abstract class base implements rehashable_driver_interface
```

Comment:
To be able to increase hashing cost factor by default. Might be used for other and future hashing types, too.